### PR TITLE
JESI-2817

### PR DESCRIPTION
**Pull request by axrs**
[JESI-2817](https://jesims.atlassian.net/browse/JESI-2817)

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -31,12 +31,8 @@
   :profiles {:dev {:plugins      [[com.jakemccrary/lein-test-refresh "0.24.1"]
                                   [lein-ancient "0.6.15"]
                                   [lein-auto "0.1.3"]
-                                  [lein-set-version "0.4.1"]
-                                  [venantius/ultra "0.6.0"]]
```
Comment (axrs) [inline project.clj:35]: Ultra should be installed per user.

https://github.com/venantius/ultra#installation

Comment (axrs) [reply project.clj:35]: Also seeing issues with latest clojure.

https://github.com/venantius/ultra/issues/108

File: project.clj [hunk 2]
```diff
@@ -31,12 +31,8 @@
...
-                   :dependencies [[circleci/circleci.test "0.4.2"]
-                                  [pjstadig/humane-test-output "0.9.0"]]
-                   :injections   [(require 'pjstadig.humane-test-output)
-                                  (pjstadig.humane-test-output/activate!)]}}
```
Comment (axrs) [inline project.clj:39]: Should be activated by user profile

https://github.com/pjstadig/humane-test-output#clojure

Also doesn't work with latest cljs

File: project.clj (hunk 1 above)
Comment (axrs) [reply project.clj:35]: Breaks new IntelliJ (2019.2) with Cursive support as well

File: project.clj (hunk 2 above)
Comment (axrs) [reply project.clj:39]: Is not compatible with Cursive, and they recommend disabling when running cursive (through env var)

File: project.clj (hunk 1 above)
Comment (axrs) [reply project.clj:35]: Ultra also does dependency re-writing for the project.clj..

**Review by BadAlgorithm — APPROVED** [review]

**Pull request merged**